app/templates/events/trackVolunteers.html

```diff
@@ -114,12 +164,61 @@ <h1 class="text-center">{{event.name}}</h1>
     <button type="submit" class="btn btn-primary float-end">Save</button>
   </div>
 </form>
+
+<!-- Outside Participant -->
+<h2 class ="text-center">Outside Participants</h2>
+<form action="#" method="post"><br>
+  <label class='d-none' id="eventLength">{{eventLength}}</label>
+  <input  class="form-control" id="eventID" name="event" value="{{event}}" type='hidden'>
+  <div class="flex-container" style="padding: 5px;">
+    <div class='form-group d-inline-flex'>
+      <a class="btn btn-success" data-bs-toggle="modal" data-bs-target="#addOutsideParticipantModal" style="margin:5px;">Add Outside Participant</a>
+</div>
+<form method="post"><br>
+  <label class='d-none' id="eventLength">{{eventLength}}</label>
+  <input  class="form-control" id="eventID" name="event" value="{{event}}" type='hidden'>
+  <table class="table" id="trackOutsideParticipants">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Email</th>
+        <th>Phone Number</th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody id="volunteerTable">
+      {% for participant in outsideParticipants%}
+      <tr>
+        <td>
+          <input
+            class="form-control"
+            type='hidden'
+            name="username{{loop.index|string}}"
+            id="{{participant.username}}"
```

Comment:
I changed the ID to the participant.email. For outside participants, email is the primary key since they do not have a bNumber. What do you think about that?